yamllint/rules/empty_values.py

```diff
@@ -73,23 +75,35 @@
 
     {a: 1, b:, c: 3}
 
+#. With ``empty-values: {forbid-key-without-values-eof: true}``
+
+   the following code snippet would **PASS**:
+   ::
+
+    key: value
+
+   the following code snippet would **FAIL**:
+   ::
+
+    key:value
+
 """
 
 import yaml
 
 from yamllint.linter import LintProblem
 
-
```

Comment:
Please don't change that

Reply:
sorry, maybe caused by autoformat in my IDE...